test/acquisition/multi_objective/test_monte_carlo.py

```diff
@@ -3,7 +3,6 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
-
```

Comment:
Can you bring this blank line back, please? Our intern linter is complaining about this.